modules/openapi-generator/src/main/resources/rust-server/Cargo.mustache

```diff
@@ -51,18 +51,19 @@ conversion = ["frunk", "frunk_derives", "frunk_core", "frunk-enum-core", "frunk-
 native-tls = { version = "0.2", optional = true }
 
 [target.'cfg(not(any(target_os = "macos", target_os = "windows", target_os = "ios")))'.dependencies]
-hyper-openssl = { version = "0.7.1", optional = true }
+hyper-openssl = { version = "0.8", optional = true }
 openssl = {version = "0.10", optional = true }
 
 [dependencies]
 # Common
+async-trait = "0.1.24"
 chrono = { version = "0.4", features = ["serde"] }
-futures = "0.1"
-swagger = "4.0"
+futures = "0.3"
+swagger = "5.0.0-alpha-1"
```

Comment:
This is an alpha release, mainly because the Hyper 0.13 code here is so new. I'll cut a 5.0 release of swagger prior to 5.0 of OpenAPI Generator being released, and update this to match.